tests/nested/manual/fakeinstaller/task.yaml

```diff
@@ -6,8 +6,8 @@ systems: [ubuntu-20.04-64, ubuntu-22.04-64]
 environment:
     # nested test so that we can test encryted installs eventually
     # TODO enable tpm variant too for testing of encryption installs
-    NESTED_ENABLE_TPM: false
-    NESTED_ENABLE_SECURE_BOOT: false
+    NESTED_ENABLE_TPM: true
+    NESTED_ENABLE_SECURE_BOOT: true
```

Comment:
I'm not sure that is enough, sadly iirc classic VMs right now will not use these variables - I can look at this when I'm back from traveling more.

Reply:
also as I commented before we want to keep the non-encrypted test as well

Reply:
I added TODOs, this will probably have to be a followup as I'm traveling right now and won't be able to work on something like spread from the train.